make/conf/jib-profiles.js

```diff
@@ -999,25 +999,36 @@ var getJibProfilesProfiles = function (input, common, data) {
         testOnlyProfilesPrebuilt["run-test-prebuilt"]["dependencies"].push(testedProfile + ".jdk_symbols");
     }
 
+    var testOnlyProfilesPrebuiltDocs = {
+        "run-test-prebuilt-docs": clone(testOnlyProfilesPrebuilt["run-test-prebuilt"])
+    };
+
+    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].dependencies.push("docs.doc_api_spec", "tidy");
+    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].environment["DOCS_JDK_IMAGE_DIR"] = input.get("docs.doc_api_spec", "install_path");
+    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].environment["TIDY"] = input.get("tidy", "home_path") + "/bin/tidy";
```

Comment:
If we are doing another round here anyway, could you try to break up these lines a bit? I'm not enforcing strict 80 chars in the js file, but this is wider than a comfortable side-by-side diff is able to render for me.

```suggestion
    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].environment["DOCS_JDK_IMAGE_DIR"]
            = input.get("docs.doc_api_spec", "install_path");
    testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].environment["TIDY"]
            = input.get("tidy", "home_path") + "/bin/tidy";
```